client/src/pages/Home.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+// import { Route } from 'react-router-dom' ;
+import { useSelector } from 'react-redux';
+import {LoggedIn, LoggedOut} from './LoggedState'
+import '../style/home.css'
+
+function NavEditor(){
+    const currentUserId = useSelector(state => state.auth.id);
+    let log = LoggedOut
+    if(currentUserId){
+        log = LoggedIn
+    }
+    return log
+}
+
+function Home(){
+    const Log = NavEditor()
+    return (
+        <>  
+            <Log />
+        </>
+    )
+}
```

Comment:
You can also use the ternary operator to conditionally render `LoggedOut` and `LoggedIn` here.